packages/terra-modal-manager/tests/wdio/modal-manager-spec.js

```diff
@@ -475,14 +475,22 @@ Terra.describeViewports('ModalManager - Disclosure Accessory', ['large'], () =>
 
   describe('Disclosure Container', () => {
     before(() => browser.url('/#/raw/tests/terra-modal-manager/modal-manager/with-disclosure-container').refresh());
+    // Skip color contrast check for elements behind a slide-shadow
+    const ignoredA11y = {
+      'color-contrast': {
```

Comment:
Can we just remove the color in the test example?

Reply:
It still might occasionally fail due to the dark grey of the transition from `slide-shadow` I think. 

We can still remove the color if we need, but I'm starting to view this to how we deal with notification-dialog where we do a similar thing and ignore contrast tests under overlay.

Reply:
Can you log an issue to terra-toolkit. If we need to define a strategy for these types of test behaviors, I'd be nice to point consumers to it as they are also probably see these same issues.

Reply:
issue logged https://github.com/cerner/terra-toolkit/issues/373